src/main/io/ledstrip.c

```diff
@@ -84,6 +84,10 @@ PG_REGISTER_ARR_WITH_RESET_FN(modeColorIndexes_t, LED_MODE_COUNT, modeColors, PG
 PG_REGISTER_WITH_RESET_FN(specialColorIndexes_t, specialColors, PG_SPECIAL_COLOR_CONFIG, 0);
 */
 
+#if defined(TARGET_CONFIG_DEFAULT_LEDSTRIP)
```

Comment:
I'm against the addition of any new TARGET_CONFIG settings. I don't think this is necessary either.